README.md

```diff
@@ -744,6 +744,12 @@ This will return the user to "https://www.example.org/logged_out.html"
 after the logout operation has completed.
 
 
+## Invalidating session
+It is possible to invalidate the current mod_auth_mellon session, without calling SLO. 
+The mod_auth_mellon cookie session will be invalidated and the session will be removed from the mod_auth_mellon cache.
+To invalidate the session, just call the endpoint "<endpoint path>/invalidate".
+
```

Comment:
Please add a warning that is session is invalidated then SLO will become impossible for that session. sorry for not asking earlier it just occurred to me that this is a side effect that users need to be aware of.
I am also wondering if this feature should be always allowed or should rather require a configure knob. I can see how some admins may not want to allow users to drop sessions without performing a full SLO.

Reply:
Yes i will add the warning.
Do you think about a toggle switch like "enabledInvalidateEndpoint"?

Reply:
yes, sounds reasonable.